_pages/en/strategy-open-first.md

```diff
@@ -0,0 +1,445 @@
+
+### Contribute
+
+> We strive to be an active member of the diverse open standards and OSS ecosystem.
+
+To be an active member of the diverse open standards and OSS ecosystem, we will continue to gain expertise to address product management and governance.
+ESDC will seek to become an active contributing member of key communities where it can add value, and while balancing costs.
+
+On the basis of the principles set out here, we will build and maintain lasting, meaningful relationships with open standards and OSS communities, building on experience from CEF, the EU-free and OSS auditing (EU-FOSSA/EU-FOSSA2) project and the ISA2 programme.
+This will contribute to the viability of the ecosystem and facilitate product management and governance.
+
+In addition to sharing source code, ESDC should consider joining open-source programme boards and committees, devise systematic methods for targeted funding, work to improve security and find ways to help critical open-source projects become self-sustainable.
+
+The GC [Guide for Contributing to OSS](https://www.canada.ca/en/government/system/digital-government/digital-government-innovations/open-source-software/guide-for-contributing-to-open-source-software.html) provides recommendations how to: verify OSS licence; verify contributing process and policies; additional approvals; contribute to the project; contribute through professional services; publish contributions regardless of upstream acceptance.
+
+### Secure
+
+> We make sure the code we use and the code we share is free from vulnerabilities by applying continuous security testing.
+
+Following IT security best practices and monitoring security advisories, automated continuous security testing will be in place to make sure that OSS components that we use in our applications are free from vulnerabilities.
+Likewise, we will thoroughly test the code that we intend to share.
+
+ESDC already has experience in this area in our security teams and the Computer Emergency Response Team for the institutions (CERT-EU), and from EU-FOSSA/EU-FOSSA2.
+
+Notes:
+
+Security and privacy methods and tools should be made available for all to ensure that this practice can be accomplished with the least barriers as possible.
+
+### Stay in control
+
+> We promote open standards and specifications that are implemented and distributed in OSS as a way to avoid vendor lock-in.
+
+Interoperability is of utmost importance to the ESDC and the GC.
+The key to interoperability is the use of well-established standards and specifications.
+To ensure our digital sovereignty and guarantee a level playing-field, for all future IT developments ESDC will promote standards and specifications that are implemented and distributed through OSS, and include this in its corporate governance approach.
+
+## 6. Implementing the Strategy
+
+The GC is an enthusiastic user of and contributor to free and OSS.
+This "Open First" strategy will ensure that ESDC becomes even more involved.
```

Comment:
I wonder if wording other than 'even more involved' would be good here... maybe 'This "Open First" strategy will ensure that ESDC increases its use of OSS and open standards and by doing so works more in alignment with GC standards and policies'?  (that's not perfect wording but something to emphasize where we are starting and where  we are going?